ui/src/app/edge/settings/profile/popover/popover.component.html

```diff
@@ -0,0 +1,29 @@
+<ion-header>
+    <ion-toolbar color="primary">
+        <ion-title>
+            Umbennen
+        </ion-title>
+        <ion-buttons slot="end">
+            <ion-button (click)="popoverController.dismiss()">
+                <ion-icon name="close-outline"></ion-icon>
+            </ion-button>
+        </ion-buttons>
+    </ion-toolbar>
+</ion-header>
+<ion-card-content>
+    <form [formGroup]="formGroup">
+        <ion-item lines="full" class="noPadding">
+            <ion-input [value]="alias.value" formControlName="alias">
```

Comment:
ist das [value] wirklich notwendig? Das sollte doch das "formControlName" übernehmen, oder? 

https://angular.io/api/forms/FormControlName: "Syncs a FormControl in an existing FormGroup to a form control element by name."

Das globale "alias" Feld wäre dann auch nicht mehr notwendig.